addons/point_of_sale/static/tests/tours/helpers/generic_components/order_widget_helpers.js

```diff
@@ -1,6 +1,6 @@
 /** @odoo-module */
 
-import { negate } from "@point_of_sale/../tests/tours/helpers/utils";
+import { negate } from "@point_of_sale/../tests/tours/helpers/utils_helpers";
```

Comment:
no need for this change. The name `utils` is clear enough

Reply:
I agree with you, this is clear. But it's not about beeing clear, but about follow the guideline. What do you think ?